testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/containers/KeycloakQuarkusServerDeployableContainer.java

```diff
@@ -176,6 +176,13 @@ protected List<String> configureArgs(List<String> args) {
     private ProcessBuilder getProcessBuilder() {
         Map<String, String> env = new HashMap<>();
         String[] processCommands = getArgs(env).toArray(new String[0]);
+        if (suiteContext.get().isAuthServerMigrationEnabled() && configuration.getImportFile() != null) {
+            for (int i = 0; i < processCommands.length; i++) {
+                if (processCommands[i].startsWith("--db-url=")) {
+                    processCommands[i]= "--db-url=\"" + processCommands[i].substring(9) + "\"";
```

Comment:
I wonder if we're not hiding from the tests too much the fact that quotes need to be used.

Reply:
@vmuzikar do you want this to be version specific somehow - check if the legacy version specifically requires double quoting?

Reply:
> do you want this to be version specific

I'm not sure I follow. :) The testsuite expects only one Keycloak version.

Reply:
This is a migration test my understanding from https://github.com/keycloak/keycloak/pull/22585#issuecomment-1695668307 is that we're initially creating an older instance.

Reply:
@vmuzikar Yes, the migration tests first start an old version (IIRC it's 19 by default), then the current version to perform the upgrade, then execute the tests. The MSSQL db needs the quotes. So we are using both type of arguments in the same test (double quotes in 19, and no double quotes in current version). As @shawkins comments we can check the old version (<23 for example), but we need something similar to this. I wouldn't do something very complicated, if the migrations tests change we can also change this part.

Reply:
Ah, sorry, I missed the migration part. Makes sense then, thanks for clarifying.